runtime/doc/lua.txt

```diff
@@ -4433,4 +4433,79 @@ tohtml.tohtml({winid}, {opt})                         *tohtml.tohtml.tohtml()*
         (`string[]`)
 
 
+==============================================================================
+Lua module: vim.net                                                  *vim.net*
+
+*vim.net.Request*
+
+    Fields: ~
+      • {wait}  (`fun(self: vim.net.Requesttimeout: integer?)`)
```

Comment:
Does someone know if this is a bug with `gen_vimdoc.lua`? Or am I doing something wrongly?

Reply:
Looks correct to me. If `Request` had other non-method field, they would also be documented here.

We could try and improve `gen_vimdoc.lua` not to do this, but at least the current behaviour is consistent.

Reply:
ah I see, that's weird. Must be a bug. Could be in the parser.